test/e2e/eventListener/eventListener_test.go

```diff
@@ -17,13 +17,12 @@ package pipeline
 
 import (
 	"context"
-	"strings"
 	"testing"
 
 	"github.com/tektoncd/cli/test/cli"
 	"github.com/tektoncd/cli/test/framework"
 	"github.com/tektoncd/cli/test/helper"
-	"gotest.tools/v3/assert"
+	"gotest.tools/assert"
```

Comment:
is this change required?

Reply:
I'm not sure of all the nuances of it, but there is an error when using `assert.Check` from `gotest.tools/v3/assert`. For what it's worth, `kn` strictly uses `gotest.tools/assert` when using `ContainsAll`.